java-client/src/main/java/org/opensearch/client/opensearch/nodes/info/NodeInfo.java

```diff
@@ -828,7 +834,7 @@ public final Builder transport(Function<NodeInfoTransport.Builder, ObjectBuilder
          * <p>
          * API name: {@code transport_address}
          */
-        public final Builder transportAddress(String value) {
+        public final Builder transportAddress(@Nullable  String value) {
```

Comment:
Please fix the double space here, or just run `./gradlew spotlessApply`
```suggestion
        public final Builder transportAddress(@Nullable String value) {
```